source/utilities.tex

```diff
@@ -9819,6 +9815,10 @@
 the expression \tcode{delete p} is well-formed and
 \tcode{Y*} is convertible to \tcode{T*}.
 
+\pnum
+\mandates
+\tcode{Y} is a complete type.
+
```

Comment:
Is this OK in light of https://github.com/cplusplus/draft/pull/3788#discussion_r383605890?

Reply:
Reverted.